.gitignore

```diff
@@ -15,3 +15,6 @@ flamegraph.svg
 # Nixos
 .direnv
 /result
+
+# RustRover
+/.idea/
```

Comment:
this is a topic for another discussion, please remove it from this PR

it might contain important data that is better to be shared between developers (like `.cargo`)

Reply:
Removed the commit.  Will raise an issue for discussion.

Reply:
See issue #434.